internal/js/modules/k6/grpc/grpc.go

```diff
@@ -86,6 +91,11 @@ func (mi *ModuleInstance) defineConstants() {
 	mustAddCode("StatusUnavailable", codes.Unavailable)
 	mustAddCode("StatusDataLoss", codes.DataLoss)
 	mustAddCode("StatusUnauthenticated", codes.Unauthenticated)
+
+	mustAddServingStatus("HealthcheckUnknown", healthpb.HealthCheckResponse_UNKNOWN)
```

Comment:
```suggestion
	mustAddServingStatus("HealthCheckUnknown", healthpb.HealthCheckResponse_UNKNOWN)
```
It seems consistent with the current API. What is the case used on JavaScript-grpc library?

Reply:
Are you referring to this https://github.com/grpc/grpc-node/blob/master/packages/grpc-health-check/src/generated/grpc/health/v1/HealthCheckResponse.ts#L6 ?

Reply:
`HealthCheck` is more consistent with the current method.